itests/paych_api_test.go

```diff
@@ -24,44 +23,38 @@ import (
 	"github.com/filecoin-project/lotus/chain/events"
 	"github.com/filecoin-project/lotus/chain/events/state"
 	"github.com/filecoin-project/lotus/chain/types"
+	"github.com/filecoin-project/lotus/itests/kit2"
 )
 
 func TestPaymentChannelsAPI(t *testing.T) {
-	kit.QuietMiningLogs()
+	kit2.QuietMiningLogs()
 
 	ctx := context.Background()
-	n, sn := kit.MockMinerBuilder(t, kit.TwoFull, kit.OneMiner)
-
-	paymentCreator := n[0]
-	paymentReceiver := n[1]
-	miner := sn[0]
-
-	// get everyone connected
-	addrs, err := paymentCreator.NetAddrsListen(ctx)
-	if err != nil {
-		t.Fatal(err)
-	}
-
-	if err := paymentReceiver.NetConnect(ctx, addrs); err != nil {
-		t.Fatal(err)
-	}
-
-	if err := miner.NetConnect(ctx, addrs); err != nil {
-		t.Fatal(err)
-	}
-
-	// start mining blocks
-	bm := kit.NewBlockMiner(t, miner)
-	bm.MineBlocks(ctx, 5*time.Millisecond)
-	t.Cleanup(bm.Stop)
+	blockTime := 5 * time.Millisecond
+
+	var (
+		paymentCreator  kit2.TestFullNode
+		paymentReceiver kit2.TestFullNode
+		miner           kit2.TestMiner
+	)
+
+	//n, sn := kit2.MockMinerBuilder(t, kit2.TwoFull, kit2.OneMiner)
+	ens := kit2.NewEnsemble(t, kit2.MockProofs()).
+		FullNode(&paymentCreator).
+		FullNode(&paymentReceiver).
+		Miner(&miner, &paymentCreator).
+		Start().
+		InterconnectAll()
+	bms := ens.BeginMining(blockTime, &miner)
```

Comment:
No need to supply the miner. If not supplied, by default we'll begin mining on all miners ;-)